lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1257,6 +1257,9 @@ def has_different_configuration(self, image):
             volume_driver=host_config.get('VolumeDriver')
         )
 
+        if kwargs.get('ignore_image', False):
```

Comment:
Why not declare a keyword argument with a default of False, up at the method signature?  That would be more explicit than a kwargs.